arrow-docs/docs/arrow/typeclasses/monoid/README.md

```diff
@@ -9,77 +9,92 @@ permalink: /arrow/typeclasses/monoid/
 
 
 
-`Monoid` extends the `Semigroup` type class, adding an `empty` method to semigroup's `combine`. The empty method must return a value that, when combined with any other instance of that type, returns the other instance, i.e.,
+`Monoid` extends the `Semigroup` type class, adding an `empty` function to semigroup's `combine`. The empty method must return a value that, when combined with any other instance of that type, returns the other instance, i.e.,
 
 ```kotlin
-(combine(x, empty) == combine(empty, x) == x)
+(a.combine(empty()) == empty().combine(a) == a)
 ```
 
-For example, if we have a `Monoid<String>` with `combine` defined as string concatenation, then `empty = ""`.
+For example, if we have a `Monoid<String>` with `combine` defined as string concatenation, then `empty() = ""`.
 
 Having an empty defined allows us to combine all the elements of some potentially empty collection of `T` for which a `Monoid<T>` is defined and return a `T`, rather than an `Option<T>` as we have a sensible default to fall back to.
```

Comment:
```suggestion
Having an empty identity allows us to combine all the elements of some potentially empty collection of `T` for which a `Monoid<T>` is defined and return a `T`, rather than an `Option<T>` as we have a sensible default to fall back to.
```

Reply:
I think it refers to having `empty()` defined

Reply:
how about:

```suggestion
Having `empty` defined allows us to combine all the elements of some potentially empty collection of `T` for which a `Monoid<T>` is defined and return a `T`, rather than an `Option<T>` as we have a sensible default to fall back to.
```

Reply:
Sounds good!